index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title> My Portfolio </title>
+	</head>
+
+	<body>
+
+		<header id="header">
+			<nav id="navigation">
+				<ul>
+					<li><a href="index.html" id="logo"> My Logo</a></li>
+					<li><a href="" class="menu-link"><img src="iconMenu.svg" alt="menu-icon" class="menu-icon"></a></li>
+				</ul>
+			</nav>
+		</header>
+		<main class="presentation">
+			<div class="greeting">
+				<h2>I’m Mariane Glad to see you!</h2>
+			</div>
+			<div class="self-description">
+				I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+				experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+			</div>
+			<a class="btn-mediaConnect" href="#">LET’S CONNECT</a>
+			<img src="social_media_icons.svg" class="media-icon" alt="">
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/fredo509/Portfolio_setup_mobile_first/pull/3#discussion_r1108911702), please consider using `ul/li` tags for adding the social media links to the portfolio 🔗 This is because apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. You can use something like below for icons 🔽

```
<ul>
  <li>
    <a href="mailto:fredostfleur23@gmail.com" rel=noopener target="_blank" class="media-link"> <img src="mailIcon.svg" alt="mail-icon"> </a>
  </li>
  <li>
    <a href="https://github.com/fredo509" rel=noopener target="_blank" class="media-link"> <img src="githubIcon.svg" alt="mail-icon"> </a>
  </li>
  ...
</ul>
```